packages/desktop-client/src/components/modals/CoverModal.tsx

```diff
@@ -105,7 +105,7 @@ export function CoverModal({
           />
           <View>
             <FieldLabel title="Cover from category:" />
-            <TapField value={fromCategory?.name} onClick={onCategoryClick} />
+            <TapField onClick={onCategoryClick}>{fromCategory?.name}</TapField>
```

Comment:
This has caused the selected value to be centre aligned on mobile instead of left as it was before.
I'm not sure if it's a blocker (personally I prefer the left aligned version) but I thought it was worth drawing attention to.

Reply:
![image](https://github.com/user-attachments/assets/84e17c2b-8bb8-4c39-9834-00a60b62e8e1)

![image](https://github.com/user-attachments/assets/670d648c-7e72-44fa-bf1c-f34623d891e6)